client/blocks/jitm/templates/home-task.jsx

```diff
@@ -0,0 +1,54 @@
+import DOMPurify from 'dompurify';
+import React from 'react';
+import TwentyPercentImageCoupon from 'calypso/assets/images/customer-home/notice-home-limited-time-coupon-20-percent.svg';
+import ThirtyPercentImageCoupon from 'calypso/assets/images/customer-home/notice-home-limited-time-coupon-30-percent.svg';
+import { NOTICE_HOME_LIMITED_TIME_OFFER_COUPON } from 'calypso/my-sites/customer-home/cards/constants';
+import Task from 'calypso/my-sites/customer-home/cards/tasks/task';
+import { recordTracksEvent } from 'calypso/state/analytics/actions';
+
+// Sanitize HTML content to allow only <b> tags and escape other tags.
+const sanitizeHTMLWithBTags = ( html ) => {
+	// Getting an instance of DOMPurify this way is needed to fix a related JEST test.
+	return DOMPurify.sanitize( html, {
+		ALLOWED_TAGS: [ 'b' ],
+	} );
+};
+
+export default function HomeTaskTemplate( { id, CTA, tracks, ...props } ) {
+	const jitmProps = { id, cta_name: id, jitm: true };
+	const tracksClickProperties = { ...jitmProps, ...tracks?.click?.props };
+
+	const illustration =
+		props.featureClass === 'limited_time_30_off_home_card'
+			? ThirtyPercentImageCoupon
+			: TwentyPercentImageCoupon;
+
+	const handleClick = () => {
+		if ( id ) {
+			recordTracksEvent?.( 'calypso_home_task_cta_click', {
+				cta_name: id,
+				...tracksClickProperties,
+			} );
+		}
+	};
+
+	return (
+		<Task
```

Comment:
Do we need to do `{ trackImpression && trackImpression() }` or something for the Tracks events to kick off here? These may come from the backend, but currently I don't see a specific Tracks event for views/clicks on this card.